src/pages/Recipes.js

```diff
@@ -31,16 +33,58 @@ export default function Recipes() {
             <Container>
                 <Row xs={1} md={4} lg={4}>
                     <Col>
-                        <Button variant="success" style={{width: '90%', height: '90%'}} onClick={() => setFilter('Dairy Free')}>Diary Free</Button>
-                    </Col>
-                    <Col>
-                        <Button variant="success" style={{width: '90%', height: '90%'}} onClick={() => setFilter('Gluten Free')}>Gluten Free</Button>
+                     {/* Dairy Free Gluten Free, Vegetarian*/}
+                        <Dropdown>
+                            <Dropdown.Toggle variant="success" id="dropdown-basic">
+                                Allergies
+                            </Dropdown.Toggle>
+
+                            <Dropdown.Menu>
+                                <Dropdown.Item variant="success" style={{width: '90%', height: '90%'}} onClick={() => setFilter('Dairy Free')}>Dairy Free</Dropdown.Item>
+                                <Dropdown.Item variant="success" style={{width: '90%', height: '90%'}} onClick={() => setFilter('Gluten Free')}>Gluten Free</Dropdown.Item>
+                                <Dropdown.Item variant="success" style={{width: '90%', height: '90%'}} onClick={() => setFilter('Vegetarian')}>Vegetarian</Dropdown.Item>
+                                <Dropdown.Item variant="success" style={{width: '90%', height: '90%'}} onClick={() => setFilter(null)}>Reset</Dropdown.Item>
+                            </Dropdown.Menu>
+                        </Dropdown>
                     </Col>
                     <Col>
-                        <Button variant="success" style={{width: '90%', height: '90%'}} onClick={() => setFilter('Vegetarian')}>Vegetarian</Button>
+                        <Dropdown>
+                            <Dropdown.Toggle variant="success" id="dropdown-basic">
+                                Food Culture
+                            </Dropdown.Toggle>
+
+                            <Dropdown.Menu>
+                                <Dropdown.Item variant="success" style={{width: '90%', height: '90%'}} onClick={() => setFilter('American')}>American</Dropdown.Item>
```

Comment:
Please use a map and add the objects instead of doing it manually. Ideally call the API to get the list of filter variants.

Reply:
FIXED